fiftyone/utils/labels.py

```
@@ -6,13 +6,21 @@
+from typing import Any, Callable, Dict, List
+import logging
+import numpy as np
 
+import fiftyone as fo
 import fiftyone.core.labels as fol
 import fiftyone.core.media as fom
 import fiftyone.core.utils as fou
 import fiftyone.core.validation as fov
 import fiftyone.utils.iou as foui
+import fiftyone.utils.utils3d as fou3d
 from fiftyone import ViewField as F
```

Ruff (0.12.2)

`fiftyone.ViewField` imported but unused

Remove unused import: `fiftyone.ViewField`

(F401)